lib/active_model_serializers/rspec/matchers/serializer.rb

```diff
@@ -0,0 +1,59 @@
+require 'active_model_serializers/test/serializer'
+
+module ActiveModelSerializers
+  module RSpec
+    module Matchers
+      module Serializer
+        extend ActiveModelSerializers::Test::Serializer
+        extend ActiveSupport::Concern
+
+        included do
+          setup :setup_serialization_subscriptions
+          teardown :teardown_serialization_subscriptions
+
+          ::RSpec::Matchers.define :use_serializer do |expected|
+            match do
+              serializer_matches?(expected)
+            end
+            failure_message do
+              message
+            end
+          end
+        end
+
+        def serializer_matches?(expectation)
+          @assert_serializer.expectation = expectation
+          @assert_serializer.message = message
+          @assert_serializer.response = response
+          @assert_serializer.matches?
+        end
+
+        def message
+          @assert_serializer.message
+        end
+
+        class Base < ActiveModelSerializers::Test::Serializer::AssertSerializer
+          def subscribe
```

Comment:
It doesn't , the behavior without that change is the one described by @rafaelgonzalez on https://github.com/rails-api/active_model_serializers/issues/1470#issuecomment-181145809. To be honest, I haven't tested this outside of my own matchers (directly on _Minitest_ using builtin assertions). I will do a report back and write a failing test.